README.md

```diff
@@ -1 +1,69 @@
-# Does-my-coding-partner-follow-JS-best-practices
\ No newline at end of file
+# Awesome-books-with-ES6
+Project requirements
+Set up a new project with webpack that is based on the webpack exercise you have already completed.
+Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
+Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
+description [string].
+completed [bool].
+index: [number].
+Write a function to iterate over the tasks array and populate an HTML list item element for each task.
+On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
+Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
```

Comment:
Kindly remember to update the contents of your README file for each project. Your README is the first file anyone who looks into your repo is going to see so having there the details from a different project is going to make it confusing for others.